tests/Redux/currentFliesReducer.test.js

```diff
@@ -0,0 +1,115 @@
+import { currentFlies } from '../../reducers/currentFlies'
+
+describe('currentFlies Reducer', () => {
+	let fliesData, newFlyData;
+
+	beforeEach(() => {
+		fliesData = [
+			{
+				"id": "8",
+				"type": "fly",
+				"attributes": {
+					"id": 8,
+					"name": "Tom",
+					"size": 4,
+					"color": "red",
+					"category": "fliessss",
+					"amount": 1
+					}
```

Comment:
Looks good. We could have a data folder that we import into these test to keep it DRY. Something to look into time permitting.